src/components/organisms/SfBottomNavigation/SfBottomNavigation.md

```diff
@@ -0,0 +1,52 @@
+# SfBottomNavigation
+
+Mobile-only bottom navigation menu with icon elements and floating Button.
+
+## HTML Template
+
+```html
+<div class="sf-bottom-navigation">
+  <div class="sf-bottom-navigation__elements" v-for="icon in icons">
+    <slot name="icon" :image="icon">
+      <div
+        class="sf-bottom-navigation__element"
+        :class="{ active: isActive }"
+        @click="isActive = !isActive"
+        v-if="icon"
+      >
+        <img
+          :src="icon"
+          alt="nav-home"
+          class="sf-bottom-navigation-element-img"
+        />
+      </div>
+    </slot>
+  </div>
+<div class="sf-bottom-navigation__element">
+      <Sf-Circle-Button class="sf-bottom-navigation-cart-button">
+        <img src="/assets/cart.svg" alt="nav-cart" class="sf-bottom-navigation-element-img"/>
```

Comment:
make this icon customizable via prop and slot (both `floating-icon`)
and make this buttonj optional (exactly like SfButton in in SfBanner